Lib/ufo2ft/featureWriters/kernFeatureWriter2.py

```diff
@@ -1,155 +1,259 @@
-"""Old implementation of KernFeatureWriter as of ufo2ft v2.30.0 for backward compat."""
```

Comment:
better to keep _some_ module-level docstring, we can remove mentinons of "old" and "backward compat" but briefly explain for posterity why the heck we have two kernFeatureWriters